bin/sirius-script-rabpm-mngr.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+import argparse
+import subprocess
+
+
+def execute_ssh(hostname, *command):
+    subprocess.run(["ssh", "-i ~/.ssh/id_ed25519_rabpm" ,f"lnls-bpm@{hostname}", *command])
+
+
+def get_hostname(rack):
+    if rack == "21":
+        return "ia-20rabpmtl-co-iocsrv"
+    elif rack.isdigit() and 1 <= int(rack) <= 20:
+        return f"ia-{rack.zfill(2)}rabpm-co-iocsrv"
+    else:
+        raise ValueError("Invalid rack number")
+
+
+def run_pcie_list(rack):
+    hostname = get_hostname(rack)
+    execute_ssh(hostname, "pcie-list-slots")
+
+
+def run_pcie_rescan(rack):
+    hostname = get_hostname(rack)
+    execute_ssh(hostname, "pcie-rescan")
+
+
+def run_pcie_remove(rack, slot):
+    hostname = get_hostname(rack)
+    execute_ssh(hostname, "pcie-remove", slot)
+
+
+def run_ioc_restart(rack, ioc, slot):
+    hostname = get_hostname(rack)
+    execute_ssh(hostname, "ioc-restart", ioc, slot)
+
+
+def run_rffe_reset(rack, virtual_slot):
+    hostname = get_hostname(rack)
+    execute_ssh(hostname, "rffe-reset", virtual_slot)
+
+
+def add_rack_arg(command):
+    command.add_argument("-r", "--rack", required=True,
+                             help="Rack number {01..20} or 21 for transport line")
```

Comment:
NIt: align `help` with the column [right after the opening parenthesis](https://peps.python.org/pep-0008/#indentation).